src/Illuminate/Queue/Worker.php

```diff
@@ -148,7 +148,7 @@ protected function daemonShouldRun()
      */
     protected function waitForChildProcess($processId, $timeout)
     {
-        declare(ticks=1) {
+        declare (ticks = 1) {
```

Comment:
> but just mentioning this because in laravel docs it says running php-cs-fixer fix should take care of correcting the code style.

Where does it say that. I thought I removed that doc because it was wrong?